sdk/test-utils/recorder/src/utils/utils.ts

```diff
@@ -115,12 +129,18 @@ export interface RegexSanitizer {
  *
  * If the body is NOT a JSON object, this sanitizer will NOT be applied.
  */
-interface BodyKeySanitizer extends RegexSanitizer {
+type BodyKeySanitizer = {
+  regex?: string;
```

Comment:
confusing to see "regex" to be boolean in some cases and a string here?

Reply:
BodyKeySanitizer is a weird special case where there isn't a string sanitizer equivalent available, so I left it untouched (calling the property "target" like we do with the other sanitizers wouldn't give any info on whether the input is supposed to be a regex or not).